### ISSUE TITLE
[change] Make email lowercase during registration

### ISSUE THREAD

**Issue by nemesisdesign** [openwisp/openwisp-radius#224]
Let's avoid the possibility of having multiple users with the same email but with different cases for letters.

We should edit the register serializer to ensure the email is converted to lowercase (+ unit test).

**Comment by sankha555** [comment]
I'll start working on this

**Comment by devkapilbansal** [comment]
@nemesisdesign shouldn't we make this change in `openwisp-users` itself?

**Comment by nemesisdesign** [comment]
@devkapilbansal I wondered about this too. It may be simpler to implement this in the `User` model. Makes sense.
@pandafy @atb00ker @sankha555 what do you think?

**Comment by sankha555** [comment]
Even I think that making this change in the app or models that deals with users might be more uniform and simpler to implement. In place of making the change in the registration of every sub-module, we can make it in one single user submodule and that can then be reflected in the other submodules consequently. I would second you in this regard @nemesisdesign @devkapilbansal .

**Comment by atb00ker** [comment]
Agreed. I like the simple solution @sankha555 made here https://github.com/openwisp/openwisp-radius/pull/228
but it would be better to ensure this for all the modules.

Moving this issue to openwisp-users! :smile:
